### ISSUE TITLE
Windows CMD prompt auto-completion 

### ISSUE THREAD

**Issue by rleonard55** [poweradminllc/PAExec#13]
Likely more of a feature request but TAB when using "paexec \remote cmd.exe" does not apear to forward the tab directly. It aphears to process it locally instead. So the auto completion does not function in the Windows CMD prompt. 

Fyi - Psexec does function this way.

**Comment by darxmorph** [comment]
That's the behavior of cmd.exe. The program can't manipulate that.

**Comment by rleonard55** [comment]
I don't think we're on the same same here. CMD does allow for tab completion. Psexec also handles this the same when launching a remote CMD session. Paexec operates differently than both, it literally injects the tab char into the command window instead.

**Comment by poweradminllc** [comment]
@rleonard55 you are correct.  I have no idea how to solve this one.  I'm open to ideas :)
